src/Form/FieldSelect.js

```diff
@@ -0,0 +1,153 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import Dropdown from '../Dropdown/Dropdown';
+
+class FieldSelect extends React.Component {
+
+  handleChange(selectedValue) {
+    this.props.handleEvent('change',
+      this.props.name, selectedValue);
+  }
+
+  hasError() {
+    let {props} = this;
+    let validationError = props.validationError;
+    return !!(validationError && validationError[props.name]);
+  }
+
+  getDropDown(dropdownItems) {
+    return (
+      <Dropdown buttonClassName="button dropdown-toggle"
+        dropdownMenuClassName="dropdown-menu"
+        dropdownMenuListClassName="dropdown-menu-list"
```

Comment:
I think we should make these classnames props so we can change them when needed.